third_party/maya/lib/usdMaya/shadingModeDisplayColor.cpp

```diff
@@ -183,24 +183,51 @@ class DisplayColorShadingModeExporter : public UsdMayaShadingModeExporter
             // instead we set the values only on the material's interface,
             // emphasizing that the interface is a value provider for
             // its shading networks.
-            UsdShadeInput dispColorIA =
-                material.CreateInput(_tokens->displayColor,
+
+            //UsdShadeInput dispColorIA = material.GetInput(_tokens->displayColor);
```

Comment:
Commented out code? Should be either removed or not.

Reply:
Ah yes should be removed, thanks!